content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-policies-cli.md

```diff
@@ -0,0 +1,199 @@
+                             ID                             |      NAME      |       DESCRIPTION        
+------------------------------------------------------------+----------------+--------------------------
+  d22957f0271ef73221005f8c11bb027888829e09/2eb7b5401728bd9f | sample-policy  | This is a sample policy  
+  d22957f0271ef73221005f8c11bb027888829e09/6c559b7225e889f9 | my-policy      | My policy                
+  d22957f0271ef73221005f8c11bb027888829e09/f7253a47e09f3bf0 | enforce-policy |                          
+```
+
+However, this output only provides some high-level information about your policies. In order to inspect the contents of a given policy, you can run the `policies view` subcommand.
+
+```sh
+chainctl policies view
+```
+
+A menu will appear, prompting you to select the policy you'd like to inspect. To avoid using the interactive menu, you can follow the `view` subcommand with the name or identification number of the policy you want to inspect.
+
+```sh
+chainctl policies view enforce-policy
+```
+
+This will output the contents of the chosen policy directly.
+
+
+## Edit policies
+
+`chainctl` includes the `policies edit` subcommand, allowing you to modify existing policies.
+
+```sh
+chainctl policies edit
+```
+
+As with `policies view`, when you run `policies edit` without any arguments it will open an interactive menu from which you can select the policy that you want to edit. You can also append the name or the identification number Chainguard Enforce uses to refer to the policy to avoid having to select the policy, as in this example.
+
+```sh
+chainctl policies edit enforce-policy
+```
+
+Regardless of how you run this command, it will open your default text editor where you can edit the specified policy's contents. After saving your changes and closing the editor, `chainctl` will prompt you to confirm whether you want to save the changes to the active policy. If you do confirm, the updated policy will be saved and applied automatically.
+
+Be aware that if you used a `.yaml` file as the basis for the policy you applied to your cluster, the `edit` subcommand will not edit the original file. It will only edit the policy as it exists within Chainguard Enforce.
+
+`chainctl` also includes the `update` subcommand. Rather than updating the policy itself, this allows you to update the description of a policy.
+
+The `policies update` subcommand requires a few arguments in order to run. Specifically, you must include the name or identification number of the policy whose description you want to update, the `--description` option, and the policy's new description. This example updates the description of `enforce-policy` policy's description to read "A description of my policy."
+
+```sh
+chainctl policies update enforce-policy --description "A description of my policy."
+```
+
+You can remove a policy's description by updating it to an empty string.
+
+```sh
+chainctl policies update enforce-policy --description ""
+```
+
```

Comment:
Maybe a closing sentence here to make the reader feel like the section is complete?

Reply:
Tried inserting one, lmk if it isn't additive enoguh